src/components/Header/Header.css

```diff
@@ -1,7 +1,36 @@
 .header {
     position: fixed;
     width: 100vw;
+	display: flex;
+	justify-content: space-between;
+	
+	
 }
 
+.logo {
+	margin-top: 10px;
+	display: flex;
+	margin-left: 30px;
+	
+}
+
+#logoText{ 
```

Comment:
generally it's better to select using `className`s instead of `id`s unless absolutely necessary for css